leap_year.py

```diff
@@ -1,8 +1,24 @@
-# Write a function `is_leap_year` that takes a year as a parameter
-#   -->**RETURNS**<-- True if the year is a leap year, False otherwise.
-# The logic-chain is somewhat similar to the Sherlock problem.
+def is_leap(year):
```

Comment:
The name of your function should be `is_leap_year`. This is likely why the automated tests are failing